lib/fitbank_api/pix/refund.rb

```diff
@@ -0,0 +1,119 @@
+# typed: strict
+# frozen_string_literal: true
+
+module FitBankApi
+  module Pix
+    # Wrapper for FitBank API used for performing PixIn refund and getting refund details
+    # Docs - https://dev.fitbank.com.br/reference/240 - GenerateRefundPixIn; https://dev.fitbank.com.br/reference/335 - GetRefundPixInById
```

Comment:
Let's split the line as it is a bit long.
```suggestion
    # GenerateRefundPixIn docs: https://dev.fitbank.com.br/reference/240
    # GetRefundPixInById docs: https://dev.fitbank.com.br/reference/335
```